lib/pages/home/pages/menu/pages/user_invites/user_invites.dart

```diff
@@ -191,7 +194,12 @@ class OBUserInvitesPageState extends State<OBUserInvitesPage> {
 
   Widget _onUserInviteDeletedCallback(
       BuildContext context, UserInvite userInvite) {
-    _refreshUser();
+    int _acceptedInvitesLength = _acceptedInvitesGroupController.getListLength();
+    int _pendingInvitesLength = _pendingInvitesGroupController.getListLength();
+    if (userInvite.createdUser != null) _user.inviteCount -= 1;
+    if (userInvite.createdUser == null) _user.inviteCount += 1;
+    if (_acceptedInvitesLength != null && _acceptedInvitesLength == 0) _acceptedInvitesGroupController.refresh();
+    if (_pendingInvitesLength != null && _pendingInvitesLength == 0) _pendingInvitesGroupController.refresh();
   }
```

Comment:
on delete, we wait till the UI list is 0, if yes then we trigger an API call to see if there are more items. Since the list API returns 10 at a time, its possible there are more items not yet loaded on the UI. (basically this helps avoid an API call on each delete)

Reply:
Was there a bug here though? If there wasn't, no reason to do this optimisations which create confusion in exchange of unnoticeable improvements.

If there was a bug, it's fine.

Reply:
Yep a small bug/bad UX... the http list apis retrieve 10 invites by default for pending/accepted. when one is deleting invites lets say we delete all first 10 (in the main page), now the UI list will be empty but there will be no more API call..n the user wont see more invites from the main page until they refresh or go back..  so to fix that , this checks when the UI list is empty on each onDelete and makes an API call to make sure we fetch results if they exist in DB. So while fixing that, we were calling the `_refreshUser` on each delete to get the latest count, i just optimised that to not make an API call always.

But as I was typing, i realised, maybe i can move the last two lines inside the group controllers.. i guess that should work too.